client/e2e/support/add-user.po.ts

```diff
@@ -0,0 +1,47 @@
+export interface TestUser {
+  name: string;
+  age: string;
+  company?: string;
+  email?: string;
+  role: 'admin' | 'editor' | 'viewer';
+}
+
+export class AddUserPage {
+  navigateTo() {
+    return cy.visit('/users/new');
+  }
+
+  getTitle() {
+    return cy.get('.add-user-title');
+  }
+
+  addUserButton() {
+    return cy.contains('button', 'ADD USER');
```

Comment:
Does this require that `ADD USER` be in all caps? Are we OK with that?

Reply:
The text in the button in the app is literally "ADD USER".

Reply:
But the test will fail if we change it to "Add User", which seems awfully fragile to me. Maybe change the test to be case insensitive?

Reply:
[It looks like you can tell `contains` to be case insensitive](https://docs.cypress.io/api/commands/contains.html#Arguments), so we could certainly do that.

Reply:
This has been fixed by using an attribute to get the button directly.